docs/advanced/data-picker.md

```diff
@@ -43,13 +43,40 @@ It is a good way to implement the widget to think that it is a component that ta
 
 ### Selector popup content
 
-Anchors in the selector can have the following attributes:
-data-close: this anchor will be ignored
-href: the url to be used for choosing
-picker-choose: when 'undefined' will load the given href url. Otherwise a choose action in the component is invoked with params: `url: href, value: picker-value, text: picker-text`.
-picker-value: the selected value
-picker-text (optional): The text to be shown in the picker button.
+**Anchors** in the selector can have the following attributes:
+- data-close: this anchor will be ignored and will close the picker
+- href: the url to be used for choosing
+- picker-choose: when is not present will, the picker will navigate as a regular anchor. Otherwise a choose action in the component is invoked with params: `url: href, value: picker-value, text: picker-text`.
```

Comment:
```suggestion
- picker-choose: when not present the picker will navigate as a regular anchor. Otherwise a choose action in the component is invoked with params: `url: href, value: picker-value, text: picker-text`.
```